src/payment/strategies/zip/zip-payment-strategy.ts

```diff
@@ -86,10 +86,9 @@ export default class ZipPaymentStrategy implements PaymentStrategy {
             try {
                 return await this._store.dispatch(this._paymentActionCreator.submitPayment({methodId: payment.methodId, paymentData: { nonce: nonce.id }}));
             } catch (error) {
-                if (error instanceof RequestError && error.body.status === 'additional_action_required') {
+                if (error instanceof RequestError && error.body.status === 'additional_action_required' && this._paymentMethod?.initializationData) {
                     return new Promise(() => {
-                        const { redirect_url } = error.body.additional_action_required.data;
-                        window.location.replace(redirect_url);
+                        window.location.replace(this._paymentMethod?.initializationData?.redirectUrl);
```

Comment:
```suggestion
                        window.location.replace(redirectUrl);
```

Reply:
Don't think this will work, since the line ` const { redirect_url } = error.body.additional_action_required.data;` is removed.